### ISSUE TITLE
Split in two different rules

### ISSUE THREAD

**Issue by nicolo-ribaudo** [atfzl/eslint-plugin-css-modules#1]
First of all, thanks for this plugin! 😃 

What do you think about splitting `css-modules/no-unused-or-extra-class` in two different rules?So that it is possible, for example, to report an error when using an undeclared class but to report a warning when a class is unused.
e.g.
```js
// .eslintrc.json
{
    ...
    "css-modules/no-undef-class": 2, // [1]
    "css-modules/no-unused-class": 1
}
```

---

**PS** It has been a little challenging to find the github repo of this package (since currently [Google indexed just the npm page](https://www.google.com/search?q=%22eslint-plugin-css-modules%22&oq=%22eslint-plugin-css-modules&aqs=chrome.1.69i57j69i59.3128j0j7&sourceid=chrome&ie=UTF-8)). Can you please add this repo's url to your `package.json` so that npm shows it?

**Comment by atfzl** [comment]
Thanks for your suggestion.
It is indeed possible to split it into two rules, will try to do it soon.

I'll add the link to the repo right away.

**Comment by atfzl** [comment]
Fixed in latest version 2.0.0.

Now there are 2 rules:

1. css-modules/no-unused-class
2. css-modules/no-undef-class

**Comment by nicolo-ribaudo** [comment]
Thank you 👍
